packages/fxa-settings/src/components/PageSecondaryEmailVerify/index.tsx

```diff
@@ -21,15 +21,15 @@ export const PageSecondaryEmailVerify = ({ location }: RouteComponentProps) => {
   const [errorText, setErrorText] = useState<string>();
   const goBack = useCallback(() => window.history.back(), []);
   const navigate = useNavigate();
+  const alertBar = useAlertBar();
   const email = (location?.state as any)?.email as string | undefined;
 
   const [verifySecondaryEmail] = useMutation(VERIFY_SECONDARY_EMAIL_MUTATION, {
     onError: (error) => {
       if (error.graphQLErrors?.length) {
         setErrorText(error.message);
```

Comment:
Hm, it may also be good in the docs to call out that when we send up user input with a mutation, that's when we need this kind of check, IE verifying a secondary email needs `if (error.graphQLErrors?.length) {` because it could return "invalid code," but signing a user out doesn't take any user input so we don't need the check for it there.

Reply:
Huh, Jared you raise a good point. From quickly looking through our test files I don't think we actually explicitly test for graphQLErrors. We just test for _an_ error. I'll document this process.